### ISSUE TITLE
converge website and blog layout

### ISSUE THREAD

**Issue by durandom** [operate-first/operate-first.github.io#80]
the blog and the website have a different layout, they don't share the same UI framework and generally have distinct publishing pipelines.

I propose we migrate the blog into the main website and use the mantine blog components - https://ui.mantine.dev/category/article-cards 

This would reduce the amount of code to be maintained and we would share the same google analytics

cc @quaid @schwesig

**Comment by durandom** [comment]
merge https://github.com/operate-first/blog to this website so that
* `/blog` points to an index of all recent blog posts (no paging needed)
* each blog post is rendered as an article card: https://ui.mantine.dev/category/article-cards
* clicking the post renders the article in the same way as the current blog site

check out https://github.com/LekoArts/gatsby-starter-minimal-blog and https://github.com/LekoArts/gatsby-themes/tree/main/themes/gatsby-theme-minimal-blog for the rendering pipeline.

**Comment by schwesig** [comment]
@traytorous Awesome, man! I will try to take a closer look. You are very fast. Thanks for the call yesterday!

**Comment by durandom** [comment]
@traytorous do you want to open a PR?

**Comment by traytorous** [comment]
@durandom @schwesig I wouldn't recommend it. This is what it would look like if you accepted my pull request. 
![operatefirsterror](https://user-images.githubusercontent.com/24616310/174311074-69bbceff-4fd4-43ff-9e8b-a584edd0fcad.png)

**Comment by durandom** [comment]
@traytorous don't worry. We won't merge the PR until it's good. I can add commits and leave comments on the PR though. So please create one and we can continue the conversation there :)

**Comment by traytorous** [comment]
It seems as though I resolved the issue. I pushed the changes. I had to lower the react version.

**Comment by traytorous** [comment]
From my what i've seen trying to implement the Added gatsby-theme-minimal-blog-core plugin seem to bring in errors. When you run it on npm run, which means its in development mode, it outputs the json data associated with the blog posting, but it also run successfully. Which means that the data is there inside of the system. However trying to build the website on my local machine will either show that the gatsby plugin brings up errors or it will build successfully. It's a hit or miss. However I did see that if you run "npm run clean" and delete the cache that it runs successfully. The next step honestly I will be trying to do is see if I can debug the issue by figuring out the gatsby-filesystem plugin. After that if that does not work I will practically have to do it manually by hand and code the entire blog feature myself. The plugin for mdx will be helpful for this.

(mdx gatsby plugin)
https://www.gatsbyjs.com/plugins/gatsby-plugin-mdx/
(the pull request that I created)
https://github.com/operate-first/operate-first.github.io/pull/85
